extension/chrome/settings/setup.ts

```diff
@@ -103,6 +104,10 @@ export class SetupView extends View {
     this.scopes = await AcctStore.getScopes(this.acctEmail);
     this.storage.email_provider = this.storage.email_provider || 'gmail';
     this.orgRules = await OrgRules.newInstance(this.acctEmail);
+    if (this.orgRules.shouldHideArmorMeta()) {
+      opgp.config.show_comment = false;
+      opgp.config.show_version = false;
+    }
```

Comment:
You could say here `if (this.orgRules.shouldHideArmorMeta() && typeof opgp !== 'undefined') {` which would address your concern (situation when opgp not imported/included).

Then you can leave the rest as is.

Reply:
Because if opgp doesn't exist, then we are at no risk of successfully sending messages - with or without comments :)